docs/dev-proc/smbios-rules.md

```diff
@@ -0,0 +1,64 @@
+# Dasharo Product Guidelines: SMBIOS Information
+
+## Introduction
+
+This document describes the guidelines for filling in the SMBIOS fields in
+devices supported by Dasharo. It is meant to be used primarily as a reference
+for Dasharo developers.
+
+## BIOS Information (Type 0)
+
+### Vendor field
+
+The vendor field is always set as follows:
+
+```
+3mdeb Embedded Systems Consulting
+```
+
+### Version field
+
+The BIOS version string is defined as follows:
+
+```bash
+Dasharo ([major_framework]+[minor_framework]) v[version]
+```
+
+- `major_framework` refers to the primary framework in which the firmware is
+  developed, e.g. `coreboot` or `edk2`.
+
+- `minor_framework` is the secondary framework, a supporting component for the
+  primary framework, e.g. a coreboot payload. May not always be used, e.g. if
+  `edk2` is used as the primary framework.
+
+*For TianoCore UEFIPayload, we use the simpler and more recognizable UEFI name.*
+
+Examples:
+
+```bash
+Dasharo (coreboot+UEFI) v1.0.0    - for coreboot-based builds with TianoCore UEFIPayload
+Dasharo (coreboot+SeaBIOS) v1.0.0 - for coreboot-based builds with SeaBIOS payload
+Dasharo (coreboot+Heads) v1.0.0   - for coreboot-based builds with Heads payload
+Dasharo (edk2) v1.0.0             - for plain edk2-based builds
+```
+
+For the version number fields, refer to [Versioning](../versioning).
```

Comment:
We should be consistent with above format:

```
- `version` means version according to official [Dasharo Versioning](../versioning) documentation.
```

Reply:
Fixed